components/public-api/gitpod/experimental/v1/oidc.proto

```diff
@@ -18,25 +18,28 @@ message OIDCClientConfig {
   // Read only.
   string id = 1;
 
-  OIDCConfig oidc_config = 2;
+  string organisation_id = 2;
```

Comment:
🙏🏻 please adjust naming to AE, so we can merge and proceed.

Reply:
done